operator/v1/types_dns.go

```diff
@@ -185,13 +186,13 @@ type DNSOverTLSConfig struct {
 	ServerName string `json:"serverName"`
 
 	// caBundle references a ConfigMap that must contain either a single
-	// CA Certificate or a CA Bundle (in the case of multiple upstreams signed
-	// by different CAs). This allows cluster administrators to provide their
+	// CA Certificate or a CA Bundle. This allows cluster administrators to provide their
 	// own CA or CA bundle for validating the certificate of upstream resolvers.
 	//
 	// 1. The configmap must contain a `ca-bundle.crt` key.
 	// 2. The value must be a PEM encoded CA certificate or CA bundle.
 	// 3. The administrator must create this configmap in the openshift-config namespace.
+	// 4. The upstream server certificate must contain a Subject Alternative Name (SAN).
```

Comment:
A SAN or a specific SAN? Why does it need a SAN, any additional context we can add?

Reply:
Excellent question. I fixed this in my latest push to note that the SAN must match `ServerName`.